File: user-service/src/main/java/rs/raf/user_service/dto/LoginRequestDto.java
package rs.raf.user_service.dto;

import lombok.Getter;
import lombok.Setter;

import javax.validation.constraints.Email;
import javax.validation.constraints.NotNull;

@Getter
@Setter
public class LoginRequestDto {
    @NotNull(message = "Email cannot be null")
    @Email(message = "Email should be valid")
    private String email;

    private String password;
}

File: user-service-kotlin/src/main/kotlin/pack/userservicekotlin/domain/dto/login/LoginRequestDto.kt
package pack.userservicekotlin.domain.dto.login

import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotNull

data class LoginRequestDto(
    @NotNull(message = "Email cannot be null")
    @Email(message = "Email should be valid")
    val email: String? = null,
    @NotNull(message = "Email cannot be null")
    val password: String? = null,
)
